File: src/test/java/cernx/colt/matrix/tfloat/impl/DenseColumnFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class DenseColumnFloatMatrix2DViewTest extends DenseColumnFloatMatrix2DTest {

    public DenseColumnFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseColumnFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseColumnFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseColumnFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tfloat/impl/DenseColumnFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.DenseColumnFloatMatrix2D
import cern.colt.matrix.MatrixTypes


class DenseColumnFloatMatrix2DViewTest(arg0: String) extends DenseColumnFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseColumnFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new MatrixTypes.DenseColumnFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new MatrixTypes.DenseColumnFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
